python_base/docker_entrypoint.sh

```diff
@@ -27,7 +27,8 @@ set -e
 if [ ! -f /virtualenv/bin/activate ]; then
     virtualenv /virtualenv -p python${INSPIRE_PYTHON_VERSION}
     source /virtualenv/bin/activate
-    pip install --upgrade pip-accel setuptools
+    pip install --upgrade pip setuptools wheel
+    cp -r /src-cache /virtualenv/src
```

Comment:
@david-caro this. So the pip install thing just does a `git pull` instead of a `git clone`